inc/namespace.php

```diff
@@ -253,14 +253,16 @@ function do_shortcode_content( $atts, string $content = '' ) : string {
 	$geos = get_all();
 
 	// If we don't have any geolocation content is empty, just bail and return an empty string.
-	if ( empty( $geos ) ) {
+	if ( empty( $geos ) || empty( $atts ) ) {
 		return '';
 	}
 
 	foreach ( $atts as $label => $value ) {
 		// Set up initial negation parameters.
 		$negate = 0;
 		$inline_negate = 0;
+		$label = $label === 'country' ? 'country-code' : $label;
+		$label = $label === 'continent' ? 'continent-code' : $label;
```

Comment:
Are lines 264-265 temporary while the old variable names are phased out?

Reply:
Not necessarily. Part of the requirement for holding onto `continent` and `country` as nouns that are supported is the shortcode itself. I don't want people to need to use [geoip country-*code*] or [geoip continent-*code*] -- I would prefer they use the expected `[geoip country]` which is easier to remember and use. To support that with the updated header names, we need to check for both things. This is especially important when we're dealing with the negations where the shortcode attribute is something like `not_country=us` -- I don't want to force people to use `not_country-code=us` which is less intuitive.